sources/packages/backend/apps/db-migrations/src/sql/SFASRestrictions/Add-col-is-included.sql

```diff
@@ -0,0 +1,6 @@
+ALTER TABLE
+  sims.sfas_restrictions
+ADD
+  COLUMN is_included bool DEFAULT FALSE;
```

Comment:
I am struggling to decide the meaningfulness of the name `is_included`.
I would suggest something like `processed` but I prefer to hear from the other DEVs. If the Team agrees that it is aligned with the purpose of the data I would be ok.

Reply:
Yes, `is_included` is definitely a bad name because doesn't give a clue about the "inclusion". I would use `imported_from_sfas` or `imported_from_integration` to avoid the SFAS name.

Reply:
I have changed this to `processed` as I too believe that it is a better column name for what we are trying to achieve. Thanks @andrewsignori-aot.